src/pages/Staking/index.tsx

```diff
@@ -168,6 +167,27 @@ const Staking = (): JSX.Element => {
   );
   useErrorHandler(voteGovernanceTokenBalanceError);
 
+  // My currently claimable rewards
+  const {
+    isIdle: currentRewardAmountIdle,
+    isLoading: currentRewardAmountLoading,
+    data: currentRewardAmount,
```

Comment:
@nud3l 
`claimableRewardAmount` could make more sense I guess.

Reply:
yeah. happy to have this changed

Reply:
I updated it.